services/app-web/src/contexts/AuthContext.tsx

```diff
@@ -39,11 +47,32 @@ function AuthProvider({
     >(undefined)
     const [loginStatus, setLoginStatus] =
         React.useState<LoginStatusType>('LOGGED_OUT')
-
+    const [isSessionExpiring, setisSessionExpiring] =
+        React.useState<boolean>(false)
+    const ldClient = useLDClient()
+    const countdownDuration =
+        ldClient?.variation(featureFlags.MODAL_COUNTDOWN_DURATION, 2) * 60
```

Comment:
Are we expecting this feature flag to be evergreen? I was expecting we would use the `timeUntilLogout` value as the default here, overridden by the feature flag only if it exists.

Reply:
I set it as a permanent flag, yeah, and timeUntilLogout gets its default value from this flag.  It gives us some flexibility to respond to user feedback around this feature, but maybe after a few months we'll want to refactor this to be hardcoded, or conditional, like you suggest.